akka-remote/src/main/scala/akka/remote/serialization/DaemonMsgCreateSerializer.scala

```diff
@@ -129,13 +132,49 @@ private[akka] class DaemonMsgCreateSerializer(val system: ExtendedActorSystem) e
       supervisor = deserializeActorRef(system, proto.getSupervisor))
   }
 
-  protected def serialize(any: Any): ByteString = ByteString.copyFrom(serialization.serialize(any.asInstanceOf[AnyRef]).get)
+  private def oldSerialize(any: Any): ByteString = ByteString.copyFrom(serialization.serialize(any.asInstanceOf[AnyRef]).get)
+
+  private def serialize(any: Any): (Int, Boolean, String, Array[Byte]) = {
+    val m = any.asInstanceOf[AnyRef]
+    val serializer = serialization.findSerializerFor(m)
+
+    // this trixery is to retain backwards wire compatibility while at the same time
+    // allowing for usage of serializers with string manifests
+    var hasManifest = false
+    val manifest = serializer match {
+      case ser: SerializerWithStringManifest ⇒
+        hasManifest = true
+        ser.manifest(m)
+      case _ if m eq null ⇒ "null"
```

Comment:
ah, it works because you fall back to the oldDeserialize, it will probably fail after changing the other comment. Or it will just work because we have a `NullSerializer` (I don't know)